src/chainparams.cpp

```diff
@@ -167,11 +167,17 @@ class CMainParams : public CChainParams {
             (0, consensus.hashGenesisBlock)
             (2500, uint256S("0x00000006dc968f600be11a86cbfbf7feb61c7577f45caced2e82b6d261d19744"))
             (15000, uint256S("0x00000000b6bc56656812a5b8dcad69d6ad4446dec23b5ec456c18641fb5381ba"))
-            (67500, uint256S("0x000000006b366d2c1649a6ebb4787ac2b39c422f451880bc922e3a6fbd723616")),
-            1487767578,     // * UNIX timestamp of last checkpoint block
-            325430,         // * total number of transactions between genesis and last checkpoint
+            (67500, uint256S("0x000000006b366d2c1649a6ebb4787ac2b39c422f451880bc922e3a6fbd723616"))
+            (100000, uint256S("0x000000001c5c82cd6baccfc0879e3830fd50d5ede17fa2c37a9a253c610eb285"))
+            (133337, uint256S("0x0000000002776ccfaf06cc19857accf3e20c01965282f916b8a886e3e4a05be9"))
+            (180000, uint256S("0x000000001205b742eac4a1b3959635bdf8aeada078d6a996df89740f7b54351d"))
+            (222222, uint256S("0x000000000cafb9e56445a6cabc8057b57ee6fcc709e7adbfa195e5c7fac61343"))
+            (270000, uint256S("0x000000000a9357513c894fb91e42e116c385da39d037c1ddb217fdd0637401f1"))
```

Comment:
Wrong blockhash: 0x00...a93575... is for block 270001.

Block 270000 is below:

2018-07-17 08:48:24 UpdateTip: new best=00000000025c1cfa0258e33ab050aaa9338a3d4aaa3eb41defefc887779a9729  height=270000  log2_work=52.732642  tx=2415465  date=2018-02-10 18:43:00 progress=0.832105  cache=6.9MiB(325tx)